app/assets/stylesheets/schools/dashboard.scss

```
@@ -7,6 +7,23 @@
     .govuk-summary-list__actions {
       width: 13%;
     }
+    .govuk-summary-list__missing {
+      border-left: 10px solid #1d70b8;
+      padding-left: 15px;
+    }
+    .govuk-summary-list__missing-heading {
+      margin-bottom: 10px;
+    }
```

Move to `assets/stylesheets/summary-list.scss`? I can see us re-using these styles on other pages than the dashboard.

Also I think we should change the prefix from `govuk-` to `app-` to make it clearer that it's app-specific styles rather than from upstream `govuk-frontend` package.

```suggestion
    .app-summary-list__missing {
      border-left: 10px solid #1d70b8;
      padding-left: 15px;
    }
    .app-summary-list__missing-heading {
      margin-bottom: 10px;
    }
```